components/collective-page/sections/Contributions.js

```diff
@@ -239,9 +248,15 @@ class SectionContributions extends React.PureComponent {
         {memberOf.length > 0 && (
           <React.Fragment>
             <ContainerSectionContent>
-              <SectionTitle data-cy="section-contributions-title" textAlign="left" mb={1}>
-                <FormattedMessage id="Contributions" defaultMessage="Contributions" />
-              </SectionTitle>
+              <SectionHeader
+                section={section}
```

Comment:
:ice_cream: I think passing the `section` down is adding complexity for not much gain. The `Contributions` section should be self-aware of its own value in the enum. So it seems fine to just do:
```suggestion
                section={Sections.CONTRIBUTIONS}
```

Something else that doesn't look consistent: for title and illustration, we expect `<SectionHeader />` to be aware of the content (it's generated based on passed `section`). But why not for `subtitle`?

My preferred approach is to keep `SectionHeader` agnostic of all content, and just do:
```jsx
<SectionHeader
  title={<FormattedMessage ... />}
  subtitle={<FormattedMessage ... />}
  illustraction={imgSrc}
/>
```

But the other approach is valid too, we should just try to be consistent.

Reply:
The intent was to have the component `SectionHeader` hold all the info for titles, subtitles, and icons eventually when we have fewer sections. However, during this transitional period, I think you're right, it should be agnostic.